kbase-extension/scss/partials/_appParams.scss

```diff
@@ -0,0 +1,17 @@
+.kb-app-params {
+  // .kb-app-params__message--advanced-hidden
+  &__message--advanced-hidden {
+    @include advanced-hidden;
+  }
+  // .kb-app-params__toggle--advanced-hidden
+  // &__toggle--advanced-hidden {
```

Comment:
All of the `(n advanced parameters (shown|hidden)) (show|hide) advanced` declarations are now styled with these classes. The toggle has the class below, text inside the toggle is the line below that (in case we want to style them differently in the future)